src/main/java/teammates/ui/controller/AdminActivityLogPageAction.java

```diff
@@ -11,33 +10,29 @@
 import teammates.common.datatransfer.FeedbackSessionAttributes;
 import teammates.common.exception.EntityDoesNotExistException;
 import teammates.common.util.ActivityLogEntry;
-import teammates.common.util.Config;
 import teammates.common.util.Const;
+import teammates.common.util.LogHelper;
 import teammates.common.util.StatusMessage;
 import teammates.common.util.TimeHelper;
 import teammates.common.util.Const.StatusMessageColor;
+import teammates.common.util.Version;
 import teammates.logic.api.GateKeeper;
 import teammates.logic.api.Logic;
 
 import com.google.appengine.api.log.AppLogLine;
-import com.google.appengine.api.log.LogQuery;
-import com.google.appengine.api.log.LogServiceFactory;
-import com.google.appengine.api.log.RequestLogs;
-import com.google.appengine.api.log.LogService.LogLevel;
 
 public class AdminActivityLogPageAction extends Action {
     
-    //We want to pull out the application logs
-    private boolean includeAppLogs = true;
     private static final int RELEVANT_LOGS_PER_PAGE = 50;
-    private static final int SEARCH_TIME_INCREMENT = 2*60*60*1000;  // two hours in millisecond
-    private static final int MAX_SEARCH_TIMES = 12;                 // maximum 1 day
+    private static final int MAX_SEARCH_TIMES = 12; // search maximum 12 times with time increment
```

Comment:
Rather than hard-coding 12, consider `(24 * 60 * 60 * 1000) / LogHelper.SEARCH_TIME_INCREMENT`. It would be resilient should you need to change the latter value as well.